### PR TITLE
build(acceptance): Fix acceptance tests (needs yarn install)

### PR DESCRIPTION
**Pull request by billyvg**

### CODE REVIEW

**Review by dcramer — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -130,6 +130,7 @@ matrix:
         - unzip ~/chromedriver_linux64.zip -d ~/
         - rm ~/chromedriver_linux64.zip
         - sudo install -m755 ~/chromedriver /usr/local/bin/
+        - yarn install --pure-lockfile
```
Comment (dcramer) [inline .travis.yml]: should 'pip inst all' have resolved this already?

Comment (billyvg) [reply .travis.yml]: why would pip resolve our js dependencies?

Comment (dcramer) [reply .travis.yml]: i thought we did something in setup.py that compiled JS assets -- maybe im mistaken

Comment (billyvg) [reply .travis.yml]: oh idk, @JoshuaRLi just refactored things, so he may have changed that, but in the meantime we need this

Comment (JoshuaRLi) [reply .travis.yml]: @dcramer are you perhaps talking about [`BuildAssetsCommand`](https://github.com/getsentry/sentry/blob/master/setup.py#L113)? IIRC everything related to building JS stuff is in the `Makefile`, for example, [this](https://github.com/getsentry/sentry/blob/master/Makefile#L135) webpack call.

Comment (JoshuaRLi) [reply .travis.yml]: Never mind, [I found the thing](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/distutils/commands/build_assets.py#L139) in setup.py that compiles JS stuff.

**Comment by JoshuaRLi** [comment]
@billyvg This works (guessing yarn is cached), but you forgot to:

```
before_install:
  - nvm install "$NPM_VERSION"
  - npm install -g "yarn@${YARN_VERSION}"
```

**Pull request merged**